server/views/icons/icon-wellcome-logo.svg

```diff
@@ -1 +1,22 @@
+	<path d="M159.447,18.494c0-7.486-3.298-11.748-10.933-11.748 c-6.077,0-10.155,4.56-10.155,10.155c0,6.411,4.634,10.043,10.785,10.043c4.374,0,8.414-1.929,9.896-6.078h-6.857 c-0.591,0.927-1.889,1.371-3.074,1.371c-2.298,0-3.558-1.556-3.744-3.744H159.447z M145.403,14.677 c0.332-2.075,1.593-3.225,3.78-3.225c1.889,0,3.225,1.446,3.225,3.225H145.403z"/>
+	<path d="M31.329,42.585c-0.445-5.299-4.745-7.783-9.783-7.783 c-5.784,0-10.416,4.04-10.416,10.118C11.13,51,15.762,55,21.546,55c5.188,0,9.301-2.927,10.004-8.153h-7.003 c-0.147,1.521-1.261,2.818-2.852,2.818c-2.82,0-3.226-2.411-3.226-4.745c0-2.335,0.406-4.782,3.226-4.782 c0.775,0,1.407,0.225,1.854,0.631c0.481,0.407,0.737,1.038,0.813,1.816H31.329z"/>
+	<path d="M43.692,44.883c0,2.038-0.369,4.782-3.149,4.782s-3.149-2.744-3.149-4.782 s0.369-4.745,3.149-4.745S43.692,42.845,43.692,44.883z M51.031,44.883c0-5.338-3.262-10.081-10.488-10.081 c-7.229,0-10.488,4.743-10.488,10.081C30.055,50.22,33.314,55,40.543,55C47.77,55,51.031,50.22,51.031,44.883z"/>
+	<rect x="50.647" y="28.056" width="7.34" height="26.464"/>
+	<rect x="59.724" y="28.056" width="7.338" height="26.464"/>
+	<path d="M87.811,46.552c0-7.488-3.3-11.75-10.934-11.75 c-6.077,0-10.152,4.559-10.152,10.155C66.725,51.369,71.357,55,77.508,55c4.373,0,8.413-1.926,9.896-6.076h-6.856 c-0.595,0.925-1.891,1.37-3.077,1.37c-2.299,0-3.559-1.557-3.745-3.742H87.811z M73.765,42.734 c0.333-2.075,1.596-3.225,3.779-3.225c1.891,0,3.226,1.447,3.226,3.225H73.765z"/>
+	<path d="M106.553,42.585c-0.443-5.299-4.743-7.783-9.784-7.783 c-5.782,0-10.412,4.04-10.412,10.118C86.357,51,90.987,55,96.769,55c5.189,0,9.304-2.927,10.008-8.153h-7.006 c-0.147,1.521-1.259,2.818-2.853,2.818c-2.819,0-3.227-2.411-3.227-4.745c0-2.335,0.407-4.782,3.227-4.782   c0.778,0,1.407,0.225,1.854,0.631c0.481,0.407,0.738,1.038,0.812,1.816H106.553z"/>
+	<path d="M114.174,29.429h-7.34v5.854h-3.26v4.521h3.26v9.193 c0,5.261,3.448,5.745,6.561,5.745c1.558,0,3.226-0.224,4.707-0.224v-5.559c-0.555,0.073-1.112,0.108-1.668,0.108 c-1.854,0-2.26-0.775-2.26-2.556v-6.709h3.928v-4.521h-3.928V29.429z"/>
+	<rect x="119.014" y="28.056" width="7.336" height="5.153"/>
+	<rect x="119.014" y="35.284" width="7.336" height="19.236"/>
+	<path d="M139.727,44.883c0,2.038-0.371,4.782-3.148,4.782 c-2.781,0-3.151-2.744-3.151-4.782s0.37-4.745,3.151-4.745C139.356,40.138,139.727,42.845,139.727,44.883z M147.067,44.883 c0-5.338-3.263-10.081-10.488-10.081c-7.228,0-10.489,4.743-10.489,10.081c0,5.337,3.262,10.117,10.489,10.117 C143.804,55,147.067,50.22,147.067,44.883z"/>
+	<path d="M146.537,54.52h7.337V44.365c0-3.075,1.669-3.706,2.93-3.706 c2.149,0,2.408,1.556,2.408,3.78V54.52h7.338V41.324c0-4.41-3.15-6.521-6.596-6.521c-2.928,0-4.894,1.002-6.267,2.926h-0.072 v-2.444h-7.078V54.52z"/>
+  <image src="/assets/images/icon-wellcome-logo.png" xlink:href="" />
```

Comment:
[This one weird trick…](https://css-tricks.com/a-complete-guide-to-svg-fallbacks/)

> To get an image fallback in SVG, without extra downloads, you need a way of including an image:
> 
> - That old browsers will recognize as valid HTML
> - That the HTML 5 parser will allow inside `<svg>`
> - That modern browsers will not interpret as an image to download.
> 
> As strange as it may seem, the SVG `<image>` element serves this purpose. The SVG `<image>` element is used to embed other image files within SVG. However, within HTML, every browser tested recognizes `<image>` as a non-standard synonym for `<img>`.